Make.defaults

```diff
@@ -22,6 +22,7 @@ DEBUGSOURCE	?= $(prefix)/src/debug/
 OSLABEL		?= $(EFIDIR)
 DEFAULT_LOADER	?= \\\\grub$(ARCH_SUFFIX).efi
 DASHJ		?= -j$(shell echo $$(($$(grep -c "^model name" /proc/cpuinfo) + 1)))
+SBATPATH	?= sbat.csv
```

Comment:
Is the intent here for shim.efi, mm.efi, and fb.efi to all always have the "shim" sbat entry?

Reply:
I think so, this was deliberately because we want a) fallback and mok manager to be allowed when SBAT is enforced and b) these to be revoked as well if a shim is revoked.

The alternative would had been to have different component names and generation number for fb and mm, but I thought that would be adding complexity for no good reason.